vendor.conf

```diff
@@ -51,7 +51,7 @@ github.com/cpuguy83/go-md2man v1.0.10
 github.com/russross/blackfriday v1.5.2
 
 # cri dependencies
-github.com/containerd/cri 0ebf032aac5f6029f95a94e42161e9db7a7e84df # release/1.3+
+github.com/containerd/cri 6ab2e10869e3e2443ae8d92d3489905bc0c987f5 https://github.com/Random-Liu/cri-containerd # master
```

Comment:
Did you mean to keep your branch?

Reply:
I don't think this should be merged until the CRI PR is merged and then this won't have to be a private tree reference. Looks like that can happen soon as the CRI PR is LGTM'ed

Reply:
Yeah, that is the plan.